### ISSUE TITLE
Doesn't work with Forge 1.13.2

### ISSUE THREAD

**Issue by Lugawi** [sp614x/optifine#2207]
Tried to use 1.13.2 E7 pre-1 with the newest forge beta it doesn't

**Comment by realjackasterisk** [comment]
#2148

**Comment by Noiwex** [comment]
Yeah, tried extracting it as MOD jar, simply not compatible.

```
Mod file Mod File: C:\Users\nx\AppData\Roaming\.minecraft\mods\OptiFine_1.13.2_HD_U_E7_pre1_MOD.jar is missing mods.toml file
```

**Comment by janissimsons** [comment]
any progress on this?

**Comment by realjackasterisk** [comment]
yes, look at the first post. #2148

**Comment by marcocorriero** [comment]
Progresses?

**Comment by realjackasterisk** [comment]
again, see the first post. #2148

@sp614x can we please close this issue? it's just a duplicate

**Comment by sp614x** [comment]
Forge compatibility added in F4_pre1 for 1.14.4.
The fix will be backported to 1.13.x.
